src/tasks/radio/radio_task.c

```diff
@@ -109,7 +111,7 @@ void radio_task_dispatch(slate_t *slate)
     // Switch to transmit mode if queue is not empty
     if (!queue_is_empty(&slate->tx_queue))
     {
-        rfm9x_transmit(&slate->radio);
+        LOG_INFO("Transmitting...");
```

Comment:
Note you're still removing the setting of radio mode to transmission here.

You can preserve the log statement, but let's add back the `rfm9x_transmit(&slate->radio)`.